lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -62,18 +89,103 @@ describe('LMSFilePicker', () => {
 
   it('fetches files when the dialog first appears', async () => {
     const wrapper = renderFilePicker();
+
     assert.calledWith(fakeApiCall, {
       authToken: 'auth-token',
       path: fakeListFilesApi.path,
     });
-    const returnedFiles = await fakeApiCall.returnValues[0];
-    // Component currently filters out Folders
-    const expectedFiles = returnedFiles.filter(file => file.type !== 'Folder');
+
+    const expectedFiles = await fakeApiCall.returnValues[0];
     wrapper.update();
+
     const fileList = wrapper.find('FileList');
     assert.deepEqual(fileList.prop('files'), expectedFiles);
   });
 
+  it('shows breadcrumbs if `withBreadcrumbs` enabled', async () => {
```

Comment:
The comment and explanatory messaging in these tests are like an air freshener sprayed to cover a code smell. I acknowledge the general stinkiness.